config/config.exs

```diff
@@ -112,6 +112,12 @@ config :ueberauth, Ueberauth,
     cognito: nil
   ]
 
+# Sentry for error tracking
+config :sentry,
+  dsn: {:system, "SENTRY_DSN"},
```

Comment:
for `application.ex` to load this, you'll need to add it to the list in `load_runtime_config()`.